Gems/EMotionFX/Code/EMotionFX/Pipeline/SceneAPIExt/Behaviors/RootMotionExtractionRuleBehavior.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <SceneAPIExt/Rules/RootMotionExtractionRule.h>
+#include <SceneAPIExt/Behaviors/RootMotionExtractionRuleBehavior.h>
+
+namespace EMotionFX::Pipeline::Behavior
+{
+    const char* RootMotionExtractionRuleBehavior::s_defaultSampleJoint = "hip";
```

Comment:
update: Would be worth adding an additional const -> `const char* const` (also might be able to be be `constexpr` too